kohttp/src/test/kotlin/io/github/rybalkinsd/kohttp/ext/HeadersExtKtTest.kt

```diff
@@ -35,3 +35,9 @@ class HeadersExtKtTest {
         }
     }
 }
+
+val Header.name
+    get() = this.first
```

Comment:
If we use plain data class (as was before) we can omit such extensions

Reply:
created #134 to implement this case